server2/Arcadia.Assistant/Arcadia.Assistant.DeviceRegistry/Program.cs

```diff
@@ -0,0 +1,48 @@
+namespace Arcadia.Assistant.DeviceRegistry
+{
+    using System;
+    using System.Diagnostics;
+    using System.Fabric;
+    using System.Threading;
+
+    using Autofac;
+    using Autofac.Integration.ServiceFabric;
+
+    using Logging;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.ServiceFabric.Services.Remoting.Client;
+
+    internal static class Program
+    {
+        /// <summary>
+        ///     This is the entry point of the service host process.
+        /// </summary>
+        private static void Main()
+        {
+            ILogger? logger = null;
+            try
+            {
+                var configurationPackage = FabricRuntime.GetActivationContext().GetConfigurationPackageObject("Config");
+
+                var builder = new ContainerBuilder();
+                builder.RegisterServiceFabricSupport();
+                builder.RegisterStatefulService<DeviceRegistry>("Arcadia.Assistant.DeviceRegistryType");
+                builder.RegisterInstance<IServiceProxyFactory>(new ServiceProxyFactory());
+                builder.RegisterServiceLogging(new LoggerSettings(configurationPackage.Settings.Sections["Logging"]));
+
+                using var container = builder.Build();
+                logger = container.ResolveOptional<ILogger>();
```

Comment:
you haven't tried to use it, have you? You must use ILogger<T> or it won't return

Reply:
As your opinion ILogger is required in service, isnt?

Reply:
github eats tags. ILogger'T. Simple ILogger won't resolve.

Reply:
Oh, caught you mind.